src/data-access/repositories/test/mongo-rundown-repository.spec.ts

```diff
@@ -70,9 +72,8 @@ describe(`${MongoRundownRepository.name}`, () => {
       const db: Db = testDatabase.getDatabase()
 
       const testee: MongoRundownRepository = createTestee({})
-      const action: () => Promise<void> = async () => testee.deleteRundown(nonExistingId)
 
-      await expect(action).rejects.toThrow(NotFoundException)
+      await expect(() => testee.deleteRundown(nonExistingId)).rejects.toThrow(NotFoundException)
```

Comment:
Is it necessary to wrap this in a function?
If it returns a promise, that should be sufficient to pass to expect.

Reply:
TIL

They do indead not need to be wrapped in a function. 
Don't know if that is because it returns a promise, or if that is just a general thing for expecting to throw.

Reply:
It is just because it is a promise/data object that you return. Throwing calls should be wrapped in lambdas.